### PR TITLE
Add several images

### PR DESCRIPTION
**Pull request by jonathanirvings**

### CODE REVIEW

**Comment by ghost** [comment]
Nice!

File: index.html [hunk 1]
```diff
@@ -68,6 +68,10 @@ <h2 id="timer">
         { src: 'img/charlize-theron.jpg', alt: "Charlize Theron" },
         { src: 'img/lisa-simpson.jpg', alt: "Lisa Simpson" },
         { src: 'img/the-captain.jpg', alt: 'The Captain' },
+        { src: 'img/greta.jpg', alt: "Greta" },
```
Comment (vt0r) [inline index.html]: How about this for a caption? ~~Also suggesting removing the leading whitespace.~~

```suggestion
        { src: 'img/greta.jpg', alt: "Greta", fullcaption: "How dare you leave your screen unlocked!" },
```

File: index.html [hunk 2]
```diff
@@ -68,6 +68,10 @@ <h2 id="timer">
...
+        { src: 'img/muhammad-sarim-akhtar.jpg', alt: "Muhammad Sarim Akhtar" },
```
Comment (vt0r) [inline index.html:71]: (addressed)

File: index.html [hunk 3]
```diff
@@ -68,6 +68,10 @@ <h2 id="timer">
...
+        { src: 'img/elon-musk.jpg', alt: "Elon Musk" },
+        { src: 'img/captain-holt.jpg', alt: "Captain Holt" },
```
Comment (vt0r) [inline index.html:74]:
```suggestion
{ src: 'img/captain-holt.jpg', alt: "Captain Holt" },
```

File: index.html [hunk 4]
```diff
@@ -82,7 +86,10 @@ <h2 id="timer">
         { src: 'img/yoda.jpg', alt: "Yoda in combat", fullcaption: "Lock your screen you must." },
         { src: 'img/upsetcat.jpg', alt: "Upset tabby cat", fullcaption: "You aren't chopped liver, lock your screen" },
         { src: 'img/orbweaver.jpg', alt: "Orb Weaver in web", fullcaption: "Lock your screen before you get tangled up" },
-        { src: 'img/skunk.jpg', alt: "Skunk in grass", fullcaption: "It stinks that you didn't lock your screen." }
+        { src: 'img/skunk.jpg', alt: "Skunk in grass", fullcaption: "It stinks that you didn't lock your screen." },
```
Comment (vt0r) [inline index.html:89]:
```suggestion
{ src: 'img/skunk.jpg', alt: "Skunk in grass", fullcaption: "It stinks that you didn't lock your screen." },
```

File: index.html [hunk 5]
```diff
@@ -82,7 +86,10 @@ <h2 id="timer">
...
+        { src: 'img/y-u-no.jpg', alt: "Y U NO", fullcaption: "Y U NO LOCK YOUR SCREEN" },
```
Comment (vt0r) [inline index.html:90]:
```suggestion
{ src: 'img/y-u-no.jpg', alt: "Y U NO", fullcaption: "Y U NO LOCK YOUR SCREEN" },
```

File: index.html [hunk 6]
```diff
@@ -82,7 +86,10 @@ <h2 id="timer">
...
+        { src: 'img/uncle-sam.jpg', alt: "Uncle Sam", fullcaption: "Uncle Sam wants you to lock your screen." },
```
Comment (vt0r) [inline index.html:91]:
```suggestion
{ src: 'img/uncle-sam.jpg', alt: "Uncle Sam", fullcaption: "Uncle Sam wants you to lock your screen." },
```

File: index.html [hunk 7]
```diff
@@ -82,7 +86,10 @@ <h2 id="timer">
...
+        { src: 'img/one-does-not-simply.jpg', alt: "One does not simply", fullcaption: "One does not simply leave your screen unlocked." }
```
Comment (vt0r) [inline index.html:92]:
```suggestion
{ src: 'img/one-does-not-simply.jpg', alt: "One does not simply", fullcaption: "One does not simply leave your screen unlocked." }
```

**Review by vt0r — APPROVED** [review]
My approval doesn't mean much here, since it's not my project, but I see no syntax errors and lots of great new pictures. :+1:

I do have ~~some very minor suggestions for uniformity, like removing leading whitespace, and I have~~ one caption idea.

File: index.html (hunk 2 above)
Comment (jonathanirvings) [reply index.html:71]: These leading spaces should be here for indentation and consistency.

File: index.html (hunk 1 above)
Comment (jonathanirvings) [reply index.html]: Added the full caption. Thanks!

File: index.html (hunk 2 above)
Comment (vt0r) [reply index.html:71]: I did the initial review from my phone, and when I looked again from the laptop, you make a very good point about the indentation! For cleanliness, I dropped the whitespace removal suggestions.

**Comment by ssaunier** [comment]
Thanks for the contribution! 🙏

**Pull request merged**